R/spew.R

```diff
@@ -80,13 +81,13 @@ call_spew <- function(base_dir, folders = NULL, data_group = "US", parallel_type
 #' @param do_subset_pums logical, do we need to subset PUMS when doing the method of sampling.  Default is TRUE.
 #' @param do_write logical, do we need to write out the synthetic populations and environments?  
 #' If FALSE, then we instead return the synthetic households and people in a list.  Default is TRUE.
-#' 
+#' @param noise a numeric value indicating the value of the noise we add to household locations during road-based sampling.  Default is .0002.
```

Comment:
I can't even read the description, because it is too long and doesn't move to a new line. This happens a lot throughout the code. Please try to keep things to ~ 80 characters per line